### PR TITLE
Add missing cbor license to LICENSE.md

### PR DESCRIPTION
**Pull request by Palantir555**
[ Description ]

- cn-cbor wasn't described in LICENSE.md. I've added it.
- I've also fixed some small markdown formatting issues to LICENSE.md
- Removed legacy `Licenses` directory
- Updated every old LMI Copyright header in the repo. Updated to 2017! Re-run the command on January to update to 2018: `find . -type f -exec sed -i 's/2003-2016. LogMeIn, Inc./2003-2017, LogMeIn, Inc./g' {} +`

[ JIRA ]

[ Reviewer ]
@DellaBitta

[ QA ]
Proofread License File.

[ Release Notes ]
Add missing 3rd party license to our LICENSE.md, removed old licenses folder, and updated all LMI copyright headers.

### CODE REVIEW

**Comment by Palantir555** [comment]
@DellaBitta @atigyi I think this is ready to be merged. Let me know if you think it's missing anything else.

File: LICENSE.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-#Copyright © 2003-2016 LogMeIn, Inc.
+# Copyright © 2003-2016 LogMeIn, Inc.
```
Comment (DellaBitta) [inline LICENSE.md]: Copyright 2017 please.

**Comment by DellaBitta** [comment]
Thanks for doing this!

Can you remove the Licenses directory, too?  The Dexonnia license contained there is already part of the main license disclosure file. If not then I'll do it later, it's ok.

**Comment by DellaBitta** [comment]
Also, approved!

File: LICENSE.md (hunk 1 above)
Comment (Palantir555) [reply LICENSE.md]: We can't label it 2018 yet, right?

There's still A LOT of 2016 headers ( https://gist.github.com/Palantir555/5e5d2b14c3f62b84fcc3c500b26e7a77 ).
We may as well wait for a week and update them all with a script.

**Comment by Palantir555** [comment]
@DellaBitta Removed the licenses folder. Check my comment about 2018 license headers; if you agree, I'll merge this right away.

**Comment by Palantir555** [comment]
@DellaBitta I've pushed all the Copyright header updates.

Command to update the headers: `find . -type f -exec sed -i 's/2003-2016. LogMeIn, Inc./2003-2017, LogMeIn, Inc./g' {} +`

Command to verify all files have been updated: `grep -ir 'copyright' . | grep -i 'logmein' | grep -i '20' | grep -v 2017`

I think this is ready to merge now. Please verify.

**Pull request merged**